packages/expo-error-recovery/src/ErrorRecovery.ts

```diff
@@ -0,0 +1,39 @@
+import ExpoErrorRecovery from './ExpoErrorRecovery';
+import { Platform } from 'react-native';
+
+let recoveredPropsToSave: String | null = null;
+let globalHandler;
+if (Platform.OS !== 'web') {
+  globalHandler = ErrorUtils.getGlobalHandler();
+  // ErrorUtils came from react-native
+  // https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/Libraries/vendor/core/ErrorUtils.js#L25
+  ErrorUtils.setGlobalHandler(async (error, isFatal) => {
+    if (ExpoErrorRecovery.saveRecoveryProps) {
+      await ExpoErrorRecovery.saveRecoveryProps(recoveredPropsToSave);
+    }
+    globalHandler(error, isFatal);
+  });
+} else {
+  globalHandler = window.onerror;
+
+  window.onerror = function() {
+    ExpoErrorRecovery.saveRecoveryProps(recoveredPropsToSave);
+    if (globalHandler) {
+      return globalHandler.aply(arguments);
```

Comment:
```suggestion
      return globalHandler.apply(arguments);
```

Did you test this code? 😅